libdnf/config.h.in

```diff
@@ -20,6 +20,7 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 #ifndef _LIBDNF_CONFIG_H_
 #define _LIBDNF_CONFIG_H_
 
+#define LIBDNF5_DISTRIBUTION_CONFIG_DIR "@CMAKE_INSTALL_PREFIX@/lib/dnf5/"
```

Comment:
Just a question - why have you used a dir and not a path to file? Anyway in the code you use automatic merge with file name.

Reply:
It was just an idea - to have cmake generate a small header file with paths to the base directories. And the details of the files in them have elsewhere.
But now the "libdnf.conf" file is defined in the middle of the source code. It would be nicer to have these things in a header file somewhere.
So I can move it here as you probably prefer.

Reply:
I made a change. So now I define the path to configuration file instead of the directory in "config.h.in".